### PR TITLE
Archive createclip changes

### PR DESCRIPTION
**Pull request by affan00**

### CODE REVIEW

File: app/assets/stylesheets/style.scss [hunk 1]
```diff
@@ -151,6 +151,14 @@ th {
   min-width: 40px;
   padding: 6px 4px;
 }
+
+.duration-error {
+  position: absolute;
+  font-size: 12px;
```
Comment (marcoherbst) [inline app/assets/stylesheets/style.scss]: Properties should be ordered position, margin-top, font-size, color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

**Comment by marcoherbst** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 2 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/evercam/evercam-dashboard/pulls/1595.

File: app/views/cameras/single.html.erb [hunk 2]
```diff
@@ -145,7 +145,8 @@
                               <tr>
                                 <td class="model-td"><label class="control-label">Duration (min):</label></td>
                                 <td>
-                                  <input id="to-date" name="to-date" class="clip-text-box form-control" type="number" min="1" max="30" step="1" value="30">
+                                  <input id="to-date" name="to-date" class="clip-text-box form-control" type="number" min="1" max="60" step="1" value="60" onkeyup="showDurationError();">
```
Comment (azharmalik3) [inline app/views/cameras/single.html.erb]: Don't use inline javascript.

File: app/views/cameras/single.html.erb [hunk 3]
```diff
@@ -259,4 +261,21 @@
   $(document).ready(function () {
     window.initializeCameraSingle();
   });
+
+  showDurationError = function () {
```
Comment (azharmalik3) [inline app/views/cameras/single.html.erb]: Move javascript to its relevant CoffeeScript file.

File: app/assets/javascripts/cameras/single/archives.js.coffee [hunk 4]
```diff
@@ -366,6 +370,17 @@ initializePopup = ->
     arrow_border: ".arrow-border"
     close: ".closepopup"
 
+refreshDataTable = ->
+  status =  jQuery.map(archives_data, (arr) ->
+    arr.status
+  )
+  if ($.inArray('Pending', status)) != -1
+    setTimeout (->
```
Comment (azharmalik3) [inline app/assets/javascripts/cameras/single/archives.js.coffee]: Assign `setTimeout` to a variable and then use that variable to clear timeout `clearTimeout(variable-name)`.

File: app/assets/javascripts/cameras/single/archives.js.coffee [hunk 5]
```diff
@@ -366,6 +370,17 @@ initializePopup = ->
...
+      archives_table.ajax.reload()
+    ), 60000
+  else
+    clearTimeout()
```
Comment (azharmalik3) [inline app/assets/javascripts/cameras/single/archives.js.coffee]: Here to use `clearTimeout(variable-name)`.

**Pull request merged**